MHVLib/MHVAsyncTask.h

```diff
@@ -62,7 +62,7 @@ typedef void (^MHVTaskCompletion) (MHVTask *task);
 //
 // This task's parent task
 //
-@property (readonly, nonatomic, weak) MHVTask *parent;
+@property (readonly, nonatomic, strong) MHVTask *parent;
```

Comment:
parent isn't run until child completes; ARC changes means the parent got deallocated before the child told it to start.